src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -227,6 +266,7 @@ def create_privacy_request(
 def get_request_status(
```

Comment:
This might be a broader conversation now that we have the concept of users but would it be a valid use case to want to audit who is making these requests?

Reply:
good future improvement, I'd probably wait to put this in unless it's specifically requested to support the frontend! I assume there's going to be more followup as we the Admin UI goes in.